deepseg.cc

```diff
@@ -82,6 +82,9 @@ cv::Mat getTensorMat(int tnum, int debug) {
 
 // deeplabv3 classes
 std::vector<std::string> labels = { "background", "aeroplane", "bicycle", "bird", "boat", "bottle", "bus", "car", "cat", "chair", "cow", "dining table", "dog", "horse", "motorbike", "person", "potted plant", "sheep", "sofa", "train", "tv" };
+// label number of "person" for DeepLab v3+ model
+const int cnum = labels.size();
+const int pers = std::find(labels.begin(),labels.end(),"person") - labels.begin();
```

Comment:
Proper C++ type usually gives `size_t` for containers. Also error checking for `std::find` returning "no match found" is missing. Last Prefer `std::distance` for retrieving the index of the `pers` label. (Internally also maps to basically this, but allos the STL to optimize.

Reply:
I fixed the `size_t` and `std::distance`.
As the `labels` list is predefined, I would not expect `find` to fail.